addons/escoria-core/game/core-scripts/save_data/esc_save_manager.gd

```diff
@@ -191,20 +193,23 @@ func load_game(id: int):
 			"res://addons/escoria-core/game/core-scripts/migrations/versions"
 		)
 		
-	# Migrate savegame through game versions
+	# Migrate savegame through game versions	
 	
-	if ProjectSettings.get_setting("escoria/main/game_version") != \
-			save_game.game_version and \
-			ProjectSettings.get_setting(
-				"escoria/main/game_migration_path"
-			) != "":
+	if escoria.project_settings_manager.get_setting(
```

Comment:
The indents here doesn't seem to fit. Lines 200/201 should at least be indentend more.